cmd/argocd/commands/plugin_test.go

```diff
@@ -0,0 +1,156 @@
+package commands
+
+import (
+	"bytes"
+	"testing"
+
+	argocdclient "github.com/argoproj/argo-cd/v2/pkg/apiclient"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// pluginPath refers to the path of the plugin found. For tests, the plugins are in testdata dir
+var pluginPath string
```

Comment:
All of the tests are mutating this value. Don't do that. Ideally each test should only depend on inputs from within its own function.

```suggestion
```

Reply:
I've fixed this now. But why is this value getting mutated?

Reply:
Ok, I read this online. Thanks!